src/xarray_regrid/regrid.py

```diff
@@ -125,26 +131,72 @@ def most_common(
 
         Args:
             ds_target_grid: Target grid dataset
-            time_dim: Name of the time dimension. Defaults to "time".
-            max_mem: (Approximate) maximum memory in bytes that the regridding routine
-                can use. Note that this is not the total memory consumption and does not
-                include the size of the final dataset. Defaults to 1e9 (1 GB).
+            expected_groups: Numpy array containing all labels expected to be in the
```

Comment:
```suggestion
            values: Numpy array containing all labels expected to be in the
```

I have never really liked `expected_groups` :) Perhaps this is better for this function.